pkg/controller/accountclaim/organizational_units_test.go

```diff
@@ -3,17 +3,284 @@ package accountclaim
 import (
 	"testing"
 
+	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/organizations"
 	"github.com/go-logr/logr"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
 	"github.com/openshift/aws-account-operator/pkg/awsclient"
 	"github.com/openshift/aws-account-operator/pkg/awsclient/mock"
+	"github.com/openshift/aws-account-operator/pkg/controller/testutils"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
+var _ = Describe("Organizational Unit", func() {
+	var (
+		nullLogger    testutils.NullLogger
+		ctrl          *gomock.Controller
+		mockAWSClient *mock.MockClient
+		r             ReconcileAccountClaim
+		ouName        = "ouName"
+		ouID          = "ouID"
+		baseID        = "baseID"
+		myID          = "MyID"
+		parentID      = "parentID"
+		awsAccountID  = "12345"
+		accountClaim  = awsv1alpha1.AccountClaim{}
+		account       = awsv1alpha1.Account{
+			Spec: awsv1alpha1.AccountSpec{
+				AwsAccountID: awsAccountID,
+			},
+		}
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mockAWSClient = mock.NewMockClient(ctrl)
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	Context("MoveAccountToOU", func() {
```

Comment:
In BDD testing, these `Context` and `It` functions should read as a sentence.

So in this case, it would be:

```go
Context("When moving an account into an OU", func() {
  It("Should return an error when it can't find the configmap", func() {
```